functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require_once 'index.php';
+
+function checkEmpty($testArray){
+    if (empty($testArray)){
+        return 'Array empty!';
+    } else {
+        return 'Items in array';
+    }
+}
+
+function checkCreateCards($monsterArray){
+    createCards($monsterArray);
+}
+
+/**
+ * @param $monsterArray
+ * @return string. This string is used to construct the cards to be displayed.
+ */
+function createCards($monsterArray) {
+    $output = "";
```

Comment:
Should make sure monsterArray is properly formatted. If not properly formatted then this function should give an error of some sort